css/sekcje/fruits.css

```diff
@@ -0,0 +1,379 @@
+  content: "";
+  box-shadow: 0 50px 24px rgba(0,0,0,0.24);
+  opacity: 0.8;
+}
+
+.fruits__price-box {
+  margin: 26px 0 24px 24px;
+  border-radius: 200px;
+  width: fit-content;
+  height: 42px;
+  background-color: var(--clr-white);
+  z-index: 999;
+}
+
+.fruits__price-value {
+  
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 1.28571;
+  letter-spacing: -0.04em;
+  color: var(--clr-red);
+  text-align: center;
+  padding: 12px 24px;
+  
+}
+
+
+.fruits__item:nth-child(1) { 
+  display: flex;
+  background-color: #f8766b;;
+  border-radius: 20px;
+}
+
+.fruits__item:nth-child(2) { 
+  display: flex;
+  background-color: #edd251;;
+  border-radius: 20px;
+}
+
+.fruits__item:nth-child(3) { 
+  display: flex;
+  background-color: #9dc262;
+  border-radius: 20px;
+}
+
+.fruits__item:nth-child(4) { 
+  display: flex;
+  background-color: #bfd663;
+  border-radius: 20px;
+}
+
+.fruits__item:nth-child(5) { 
+  display: flex;
+  background-color: #e93f29;
+  border-radius: 20px;
+  }
+
+.fruits__item:nth-child(6) { 
+  display: flex;
+  background-color: #db4f4b;
+  border-radius: 20px;
+}
+
+.fruits__item:nth-child(7) { 
+  display: none;
+  background-color: #9b5366;
+  border-radius: 20px;
+}
+
+.fruits__item:nth-child(8) { 
+  display: none;
+  background-color: #f47f4c;
+  border-radius: 20px;
+}
+
+.fruits__item:nth-child(9) { 
+  display: none;
+  background-color: #98b37e;
+  border-radius: 20px;
+}
+
+.fruits__btn-div {
+  display: flex;
+  justify-content: center;;
+  margin-top: 44px;
+}
+
+/* Fruits button */
+
+.fruits__btn {
+  background-color: #9dc262;
+  color: var(--clr-white);
+  transition: .3s;
+  width: 80px;
+  height: 80px;
+  border-radius: 100%;
+  border: none;
+  fill: var(--clr-white);
+  position: relative
+}
+
+.fruits__order {
+  position: absolute;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 1.28571;
+  letter-spacing: -0.04em;
+  text-transform: uppercase;
+  text-align: center;
+  text-shadow: 0 4px 70px 0 rgba(0,0,0,0.1);
+  transform: rotate(-14deg);
+  left: 35%;
+  top: 40%;
+}
+
+.fruits__btn-icon {
+  position: absolute;
+  left: 10%;
+}
+
+.fruits__btn:hover {
+  animation: pulse 1s infinite;
+  transition: .3s;
+  background-color: var(--clr-white);
+  color: #9dc262;
+  fill: #9dc262;
+}
+
+.fruits__btn:active {
+   background-color: var(--clr-white);;
+  color: #9dc262;
+  fill: #9dc262;
+  text-shadow: 0px 0px 4px var(--clr-white);
+  box-shadow: 0 0 10px var(--clr-white), 0 0 40px var(--clr-white), 0 0 80px var(--clr-white);
+  transition-duration: 0.1s;
+}
+
+@keyframes pulse {
+  0% {
+    transform: scale(1);
+  }
+  70% {
+    transform: scale(.9);
+  }
+    100% {
+    transform: scale(1);
+  }
+}
+
+/* SECTION TABLET */
+
+/* Fruits text */
+
+@media 
+(min-width: 768px) {
+
+.container {
+max-width: 768px;
+}
+
+.fruits__text {
+  width: 624px;
+  margin: 144px auto 44px auto;
+}
+
+.fruits__title {
+  font-size: 40px;
+line-height: 1.1;
+}
+
+.fruits__discr {
```

Comment:
To będzie niepotrzebne po nałożeniu powyższych poprawek

Reply:
Solved